static/js/task/controllers/task.controller.js

```diff
@@ -58,16 +58,7 @@
 
                     self.taskData = data[0].data;
                     self.taskData.id = self.taskData.task ? self.taskData.task : id;
-                    angular.forEach(self.taskData.task_template.template_items, function(obj, index){
-                        var tabIndex;
-                        if(obj.role === "display"){
-                            tabIndex = -1;
-                        }
-                        else {
-                            tabIndex = $filter('filter')(self.taskData.task_template.template_items.slice(0, index), {'role': 'input'}).length + 1;
-                        }
-                        angular.extend(obj, {tabIndex: tabIndex});
-                    });
+
```

Comment:
We only add tabIndex to components which need to be excluded from tabs. The rest of the components follow the default indexing which works as intended with above code.